Source/ParameterEncoding.swift

```diff
@@ -241,8 +241,11 @@ public struct URLEncoding: ParameterEncoding {
             let value = parameters[key]!
             components += queryComponents(fromKey: key, value: value)
         }
-
+        #if swift(>=4.0)
+        return components.map { "\($0.0)=\($0.1)" }.joined(separator: "&")
+        #else
         return components.map { "\($0)=\($1)" }.joined(separator: "&")
+        #endif
```

Comment:
Addressed in [a0337](https://github.com/Alamofire/Alamofire/pull/2163/commits/03dbf047a5fd956e850a135efbf651c390ea0337).